### PR TITLE
[improvement](error info)improve the s3 path err msg

### PR DESCRIPTION
**Pull request by zy-kkk**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [x] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ExportStmt.java [hunk 1]
```diff
@@ -244,7 +244,7 @@ public static String checkPath(String path, StorageBackend.StorageType type) thr
             }
         } else if (type == StorageBackend.StorageType.S3) {
             if (schema == null || !schema.equalsIgnoreCase("s3")) {
-                throw new AnalysisException("Invalid export path. please use valid 'S3://' path.");
+                throw new AnalysisException("Invalid export path. please use valid 's3://' path.");
```
Comment (cambyzju) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/ExportStmt.java:247]: !schema.equalsIgnoreCase("s3")

why we need change 'S3' to 's3'?

Comment (zy-kkk) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/ExportStmt.java:247]: When the s3 path entered by the user is wrong, we should return the correct prompt information, and returning 'S3' is wrong, so I changed to the correct 's3'

**Review by morningman — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**